src/components/TodoItem/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import users from '../../api/users';
+import './TodoItem.css';
+
+export default class TodoItem extends Component {
+  state = {
+    usersForTask: users,
+    completed: this.props.completed,
+  }
+
+  toogleCompleted = (e) => {
+    e.preventDefault();
+    this.setState(state => ({ completed: !state.completed }));
+  }
+
+  render() {
+    const { title, userId = 0 } = this.props;
+    const { usersForTask, completed } = this.state;
+    const user = usersForTask.find(person => person.id === userId) || '';
+    const todoItemClass = classNames('todo-item ', { ' completed': completed });
```

Comment:
```suggestion
    const todoItemClass = classNames('todo-item ', { 'completed': completed });
```
there is no need to add space